src/ZenstruckFoundryBundle.php

```diff
@@ -244,6 +244,11 @@ public function process(ContainerBuilder $container): void
         }
     }
 
+    public function shutdown(): void
+    {
+        Configuration::shutdown();
+    }
```

Comment:
prevents some problems where we boot Symfony's kernel before the tests: foundry was keeping its "booted" stated, wheras it should have been shutdown